scripts/cc-cli/src/test/parsing.test.ts

```diff
@@ -0,0 +1,146 @@
+import {
```

Comment:
This is not hooked up into CI, see for example:
https://github.com/gluwa/creditcoin/blob/dev/.github/workflows/ci.yml#L280

Note: as part of #1165 the `integration-test` job is being renamed. Please use a more appropriate name! 

Note2: I realize these aren't "integation" type tests as of yet so naming is up to you. We'll adjust it later anyway.